src/context-items/rename-field.ts

```diff
@@ -0,0 +1,28 @@
+import { CustomValidator, Meta } from '../base';
+import { Context } from '../context';
+import { ContextItem } from './context-item';
+
+export class RenameFieldContextItem implements ContextItem {
+  message: any;
```

Comment:
When you move `rename` out of `Validators`, I think that this won't be necessary anymore